tests/model_serving/model_server/kueue/test_kueue_isvc_serverless.py

```
@@ -0,0 +1,126 @@
+                "namespace_selector": {},
+            },
+            {"name": resource_flavor_name},
+            {"name": local_queue_name, "cluster_queue": cluster_queue_name},
+        )
+    ],
+    indirect=True,
+)
+class TestKueueInferenceServiceServerless:
+    """Test inference service with serverless deployment"""
+
+    def test_kueue_inference_service_model_mesh(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)